### DISCUSSION TITLE
Writing error from TMP to DST

### DISCUSSION THREAD

**Discussion by TaijiMonster** [Chia-Network/chia-blockchain#6291] [Support]
I am using a S3 bucket (via goofys) and gets the following error:
Could not copy "/tempchiaraid0/plot-k32-2021-05-31-20-24-884cd963a00856fc8428b721cabacdaf343619119b7f16c28cab948324fbb9f0.plot.2.tmp" to "/xxx/chia-pc1/plot-k32-2021-05-31-20-24-884cd963a00856fc8428b721cabacdaf343619119b7f16c28cab948324fbb9f0.plot.2.tmp". Error Operation not permitted. Retrying in five minutes.

I am able to use non-roor user to create directory/write file into the DST directory specified with no issue, I can see that plotman/chia written the file but it's 0 bytes.

![image](https://user-images.githubusercontent.com/16340329/120259547-bf13ef80-c2c6-11eb-9e53-8ae6f8bfb513.png)

Please help....

**Comment by BrandtH22** [comment]
@TaijiMonster , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the permission issue is restricting the clients ability to move the temp plot files to the destination drive which would likely be due to permissions on that drive needing an update.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
